exotations/task_maps/task_map/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_definitions(-DEXOTICA_DEBUG_MODE)
 
 find_package(catkin REQUIRED COMPONENTS
   exotica
+  exotica_py
```

Comment:
You don't need ```task_map``` to build ```exotica_py```. The package is a plugin so you can still use it without linking to it. It scales if you add new packages. Otherwise you would have to make ```exotica_py``` depend on all packages in c++. This way you only import them in python when needed.
```task_map_py``` only adds specification of new classes in python but you can still use the abstract interface of ```TaskMap``` (defined in ```exotica```). In most cases that is all you want to do anyway.